tests/test_arraycreation.py

```diff
@@ -110,3 +110,29 @@ def test_loadtxt(type):
         dpnp_res = dpnp.loadtxt(fh, dtype=type)
 
         numpy.testing.assert_array_equal(dpnp_res, np_res)
+
+
+@pytest.mark.parametrize("k",
+                         [-6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6],
+                         ids=['-6', '-5', '-4', '-3', '-2', '-1', '0', '1', '2', '3', '4', '5', '6'])
```

Comment:
do you really need to test all these cases?
Isn't one instance enough for cases `k>0`, `k=0`, `k<0` for testing?